DuckDuckGo/DuckPlayer/YoutubeOverlayUserScript.swift

```diff
@@ -164,15 +164,18 @@ extension YoutubeOverlayUserScript {
         
         switch pixelName {
         case "play.use":
-            Pixel.fire(pixel: Pixel.Event.duckPlayerViewFromYoutubeViaMainOverlay)
+            Pixel.fire(pixel: Pixel.Event.duckPlayerViewFromYoutubeViaMainOverlay, debounce: 2)
```

Comment:
Debounce is needed on this pixels as these come directly from the UI and I've seen dupes some times.